packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -117,7 +115,7 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
     }
 
     modifier notEmergencyShutdown() {
-        require(emergencyShutdownTimestamp == 0, "Contract emergency shutdown");
+        _notEmergencyShutdown();
```

Comment:
this was inconsistent. it should call the internal method `_notEmergencyShutdown`